modules/lazy-images/lazy-images.php

```diff
@@ -81,6 +83,15 @@ public function add_image_placeholders( $content ) {
 		return $content;
 	}
 
+	/**
+	 * Processes images in content by acting as the preg_replace_callback
+	 *
+	 * @since 5.6.0
```

Comment:
Maybe should be 5.7 now

Reply:
That method was actually added in 5.6. I'm just now added a docblock though.

Reply:
hahaha great. Thanks!